tests/cypress/e2e/session.cy.js

```diff
@@ -0,0 +1,111 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+describe("Session", () => {
+    it('can login', () => {
+        cy.blockGLPIDashboards();
+
+        // Go to login page
+        cy.visit('/', {
+            headers: {
+                'Accept-Language': 'en-GB,en;q=0.9',
+            }
+        });
+        cy.title().should('eq', 'Authentication - GLPI');
+
+        // Fill form
+        cy.findByRole('textbox', {'name': "Login"}).type('e2e_tests');
+        cy.findByLabelText("Password").type('glpi');
+        cy.findByRole('checkbox', {name: "Remember me"}).check();
+        cy.getDropdownByLabelText("Login source").selectDropdownValue('GLPI internal database');
+
+        // Submit, the url should now contain /front/central.php or /front/helpdesk.public.php
+        cy.findByRole('button', {name: "Sign in"}).click();
+        cy.url().should('match', /\/front\/(central|helpdesk.public).php/);
+
+        // Validate cookies
+        cy.getCookies().should('have.length.gte', 2).then((cookies) => {
+            // Should be two cookies starting with 'glpi_' and one of them should end with '_rememberme'
+            expect(cookies.filter((cookie) => cookie.name.startsWith('glpi_'))).to.have.length(2);
+            expect(cookies.filter((cookie) => cookie.name.startsWith('glpi_') && cookie.name.endsWith('_rememberme'))).to.have.length(1);
+        });
+    });
+
+    it('can logout', () => {
+        // Login and go to any page
+        cy.login();
+        cy.visit('/front/computer.form.php');
+
+        // Logout
+        cy.findByRole('link', {name: 'User menu'}).click();
+        cy.findByRole('link', {name: 'Logout'}).click();
+
+        // Should be redirected to login page
+        cy.visit('/front/computer.form.php');
+        cy.findByText('Your session has expired. Please log in again.').should('exist');
+    });
+
+    it("redirect to requested page after login", () => {
+        // Must visit twice because glpi doens't support redirect on the first
+        // ever visit due to some cookies checks...
+        cy.visit('/front/ticket.form.php');
+        cy.visit('/front/ticket.form.php', {
+            failOnStatusCode: false
+        });
+        cy.findByRole('link', {'name': "Log in again"}).click();
+
+        // Login as e2e_tests
+        cy.findByRole('textbox', {'name': "Login"}).type('e2e_tests');
+        cy.findByLabelText("Password").type('glpi');
+        cy.findByRole('button', {name: "Sign in"}).click();
+
+        // Should be redirected to requested page
+        cy.url().should('contains', "/front/ticket.form.php");
+    });
+
+    it("can change profile", () => {
```

Comment:
Maybe you can check the presence of the "Administration" menu entry too.

Reply:
I can but "Observer" can see it too (all our default profiles can except self-service).
I'll switch to self service so we can validate that the menu is gone.